supervisory-authorities/derfdw.json

```
@@ -0,0 +1,16 @@
+{
+    "slug": "derfdw",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Rundfunkdatenschutzbeauftragter Deutsche Welle",
+    "address": "Kurt-Schumacher-Straße 3\n53113 Bonn\nDeutschland",
+    "phone": "+49 228 4290",
+    "email": "datenschutz@dw.com",
+    "web": "https://www.dw.com/",
```

Not great but at least a little more specific (that's what's linked on: https://www.rundfunkdatenschutzkonferenz.de/kontakt).

```suggestion
    "web": "https://www.dw.com/de/datenschutzbericht/a-16130989",
```